var/spack/repos/builtin/packages/ascent/package.py

```diff
@@ -35,22 +35,21 @@ class Ascent(Package, CudaPackage):
 
     homepage = "https://github.com/Alpine-DAV/ascent"
     git      = "https://github.com/Alpine-DAV/ascent.git"
-    url      = "https://github.com/Alpine-DAV/ascent/releases/download/v0.5.0/ascent-v0.5.0-src-with-blt.tar.gz"
+    url      = "https://github.com/Alpine-DAV/ascent/releases/download/v0.5.1/ascent-v0.5.1-src-with-blt.tar.gz"
 
     maintainers = ['cyrush']
 
     version('develop',
             branch='develop',
-            submodules=True)
-
-    version('0.5.0', sha256='2837b7371db3ac1bcc31a479d7cf0eb62a503cacadfa4187061502b3c4a89fa0')
```

Comment:
Why remove the stable release? We usually like to have at least one stable release that gets installed by default since develop is a moving target.

Reply:
stable release will no longer build after these changes to spack. Also we are not supporting earlier releases, given the pace at which bug fixes and features are being put in.

Reply:
also on this topic: Making a stable release work require another pass at combos of variants and versions, exploding the depends on logic more.  Also, we have to use a custom version of VTK-m. This dep has lots of API changes, we aren't able to use tagged releases or develop, we have to dial a specific version in. VTK-m folks are ok with us setting a custom version for use in ascent in their spack package, but they don't want these to start piling up.